src/encoded/schemas/antibody.json

```diff
@@ -80,13 +80,19 @@
         }
     },
     "facets": {
-        "antibody_vendor" : {
-            "title" : "Vendor"
+        "antibody_vendor.display_title" : {
+            "title": "Vendor"
+        },
+        "antibody_target.display_title": {
+            "title": "Target"
         }
     },
     "columns": {
         "antibody_vendor": {
```

Comment:
Is not a similar change required here?

Reply:
Surprisingly it works without that change (not sure why though).

Reply:
I suspect these have never worked, but it's not something used too often to really notice.

Reply:
I had thought that the default when a property was not included was to use display_title and apparently that works for columns but I guess not for facets.